com.civicrm.hrjobroles/tests/phpunit/HrJobRolesTestBase.php

```diff
@@ -1,130 +0,0 @@
-<?php
```

Comment:
Can you add some note to the PR description for why this test was deleted

Reply:
Will do. I didn't do it before because I thought the [commit](https://github.com/civicrm/civihr/pull/2382/commits/66d41e97676cf87eb90b519f5e3fd456b782bbd4) message what descriptive enough